guide/02-api-overview/release-notes.ipynb

```diff
@@ -1,5 +1,438 @@
 {
```

Comment:
<p><a href="https://developers.arcgis.com/python/api-reference/arcgis.raster.toc.html#raster" target="_blank">Raster</a></p><ul><li>create Raster objects from stac catalogs with&nbsp;<code><a href="https://developers.arcgis.com/python/api-reference/arcgis.raster.toc.html#arcgis.raster.Raster.from_stac_item" target="_blank">from_stac_item(</a>)</code></li></ul><p>Can it be changed to </p><ul><li>create Raster objects from <strong>stac items</strong> with <a href="https://developers.arcgis.com/python/api-reference/arcgis.raster.toc.html#arcgis.raster.Raster.from_stac_item" target="_blank">from_stac_item(</a>)</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1026/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
Done!